src/java/htsjdk/variant/variantcontext/GenotypeJEXLContext.java

```diff
@@ -0,0 +1,58 @@
+package htsjdk.variant.variantcontext;
+
+import htsjdk.variant.vcf.VCFConstants;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ * @author bbimber
+ *
+ * implements the JEXL context for Genotype; this saves us from
+ * having to generate a JEXL context lookup map everytime we want to evaluate an expression.
+ *
+ */
+public class GenotypeJEXLContext extends VariantJEXLContext {
+    private Genotype g;
+
+    private interface AttributeGetter {
+        public Object get(Genotype g);
+    }
+
+    private static Map<String, AttributeGetter> attrs = new HashMap<String, AttributeGetter>();
+
+    static {
+        attrs.put("g", new AttributeGetter() { public Object get(Genotype g) { return g; }});
+        attrs.put(VCFConstants.GENOTYPE_KEY, new AttributeGetter() { public Object get(Genotype g) { return g.getGenotypeString(); }});
+
+        attrs.put("isHom", new AttributeGetter() { public Object get(Genotype g) { return g.isHom() ? "1" : "0"; }});
```

Comment:
FWIW, the checked in project IPR file has the languageLevel as 1.6.  If you're officially on 1.8, it might be nice to update that file (not in this PR).